File: DSA/NiharikaChand_590017697/Day4Question1.java
import java.util.*;
class Day4Question1 
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size: ");
        int n = sc.nextInt();
        int nums[] = new int[n];
        int i;
        for(i=0; i<n; i++)
        {
            System.out.println("Enter an element: ");
            nums[i] = sc.nextInt();
        }

        int min = Integer.MAX_VALUE;
        for(i=0; i<n; i++)
        {
            if(nums[i] < min)
            min = nums[i];
        }
        System.out.println("Minimum element is: " + min);
    }
}

File: DSA/GEHNA_KHANNA_590013140/day4question1.c
#include <stdio.h>
int main() 
{
    int n;
    printf("Enter array size:\n");
    scanf("%d", &n);

    int nums[n];
    printf("Enter array elements:\n");
    for (int i = 0; i < n; i++) {
        scanf("%d", &nums[i]);
    }
    int min = nums[0];
    for (int i = 1; i < n; i++) 
    {
        if (nums[i] < min) 
        {
            min = nums[i];
        }
    }
    printf("The minimum element in the array is: %d\n", min);
    return 0;
}
